scripts/lib/CIME/case.py

```diff
@@ -908,6 +908,8 @@ def configure(self, compset_name, grid_name, machine_name=None,
             else:
                 self._check_testlists(compset_alias, grid_name, files)
 
+        self.set_value("REALUSER", os.environ["USER"])
```

Comment:
this is kinda ugly - who is real user?  The current user, or the one that setup the case?

Reply:
Yeah, it isn't great. REALUSER is the one who created the case. This fixes the bless_test_results issue we were discussing yesterday without needing additional flags to that tool.